layers/+emacs/org/packages.el

```diff
@@ -810,3 +811,29 @@ Headline^^            Visit entry^^               Filter^^                    Da
     :defer t
     :init
     (add-hook 'org-mode-hook 'org-sticky-header-mode)))
+
+(defun org/init-verb ()
+  (use-package verb
+               :after org
+               :config
```

Comment:
Use `:defer t` to defer the loading.
Keybindings should generally go inside `:init`. Registering keybindings adds entrypoints to the package but doesn't load the package. 
I'm not sure if `:after` is required but keep it if removing it is causing an issue.

Reply:
package is now properly defered.